blocks/multiplatform/hero/index.tsx

```diff
@@ -0,0 +1,54 @@
+import cn from 'classnames';
+
+import { useTextStyles } from '@rescui/typography';
+import { Button } from '@rescui/button';
+
+import { useML } from '@jetbrains/kotlin-web-site-ui/out/components/breakpoints-v2';
+
+import android from './icons/android.svg';
+import ios from './icons/ios.svg';
+import web from './icons/web.svg';
+import desktop from './icons/desktop.svg';
+import server from './icons/server.svg';
+
+import logo from './logo.svg';
+
+import styles from './hero.module.css';
+
+const platforms = [
```

Comment:
JIC: We have some platform types declared in the case studies module. Probably it's worth extracting it to some common place (or maybe not)

Anyway, there is a difference in names for platforms on different pages (frontend + backend on case studies, web + server on this kmp landing) and there is still a discussion what options are better.
Not sure if there are any certain action points here, so just to share some context